doc/releases/changelog-0.41.0.md

```diff
@@ -420,65 +420,85 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
 
 <h4>QROM State Preparation 📖</h4>
 
-* Added template `qml.QROMStatePreparation` that prepares arbitrary states using `qml.QROM`.
-  [(#6974)](https://github.com/PennyLaneAI/pennylane/pull/6974)
+Quantum Read-Only Memory (QROM) isn't just useful for loading bitstrings.
+It can be leveraged to prepare entire quantum states!
 
-<h4>Dynamical Lie Algebras 🕓</h4>
+* Added template :class:`qml.QROMStatePreparation <pennylane.QROMStatePreparation>` that prepares arbitrary states using `qml.QROM`.
+  [(#6974)](https://github.com/PennyLaneAI/pennylane/pull/6974)
 
-* Created a new `qml.liealg` module for Lie algebra functionality.
+  ```python
+  state_vector = np.array([0.5, -0.5, 0.5, 0.5])
 
-  `qml.liealg.cartan_decomp` allows to perform Cartan decompositions `g = k + m` using _involution_ functions that return a boolean value.
-  A variety of typically encountered involution functions are included in the module, in particular the following:
+  dev = qml.device("default.qubit")
+  wires = qml.registers({"work_wires": 1, "prec_wires": 3, "state_wires": 2})
 
-  ```
-  even_odd_involution
-  concurrence_involution
-  A
-  AI
-  AII
-  AIII
-  BD
-  BDI
-  DIII
-  C
-  CI
-  CII
+  @qml.qnode(dev)
+  def circuit():
+      qml.QROMStatePreparation(
+          state_vector, wires["state_wires"], wires["prec_wires"], wires["work_wires"]
+      )
+      return qml.state()
   ```
 
   ```pycon
-  >>> g = qml.lie_closure([X(0) @ X(1), Y(0), Y(1)])
-  >>> k, m = qml.liealg.cartan_decomp(g, qml.liealg.even_odd_involution)
-  >>> g, k, m
-  ([X(0) @ X(1), Y(0), Y(1), Z(0) @ X(1), X(0) @ Z(1), Z(0) @ Z(1)],
-   [Y(0), Y(1)],
-   [X(0) @ X(1), Z(0) @ X(1), X(0) @ Z(1), Z(0) @ Z(1)])
+  >>> print(circuit()[:4].real)
+  [ 0.5 -0.5  0.5  0.5]
   ```
 
-  The vertical subspace `k` and `m` fulfil the commutation relations `[k, m] ⊆ m`, `[k, k] ⊆ k` and `[m, m] ⊆ k` that make them a proper Cartan decomposition. These can be checked using the function `qml.liealg.check_cartan_decomp`.
+<h4>Dynamical Lie Algebras 🕓</h4>
 
-  ```pycon
-  >>> qml.liealg.check_cartan_decomp(k, m) # check Cartan commutation relations
-  True
-  ```
+The new :mod:`qml.liealg <pennylane.liealg>` module provides a variety of Lie algebra functionality.
 
-  `qml.liealg.horizontal_cartan_subalgebra` computes a horizontal Cartan subalgebra `a` of `m`.
+:func:`qml.liealg.cartan_decomp <pennylane.liealg.cartan_decomp>` allows performance of Cartan decompositions `g = k + m` using _involution_ functions that return a boolean value.
```

Comment:
I recommend that we have a bulleted list of features after the first sentence above 👀